website/server/libs/taskManager.js

```diff
@@ -284,3 +295,213 @@ export function moveTask (order, taskId, to) {
     order.splice(to, 0, taskId);
   }
 }
+
+/**
+ * Scores a task.
+ * @param user the user that is making the operation
+ * @param task The task to score
+ * @param direction "up" or "down" depending on how the task should be scored
+ *
+ * @return Response Data
+*/
+async function scoreTask (user, task, direction, req, res) {
```

Comment:
now that these methods have been extracted in a library it might be a good idea to cover them with unit tests